packages/warriorjs-cli/templates/readme/abilities.ejs

```diff
@@ -1,6 +1,6 @@
 <%_
-  const actions = abilities.filter(ability => ability.action);
-  const senses = abilities.filter(ability => !ability.action);
+  const actions = abilities.filter(ability => ability.action).sort((a, b) => a.name > b.name);
+  const senses = abilities.filter(ability => !ability.action).sort((a, b) => a.name > b.name);
```

Comment:
Can you refactor this, extracting the sort call and saving the result in a new variable?

Reply:
@olistic Sounds good, I was already thinking it might be better to do this just once, will pick this up today/tomorrow!